pkg/reconcile/pipeline/infinispan/handler/manage/conditions.go

```diff
@@ -50,6 +40,11 @@ func PreliminaryChecks(i *ispnv1.Infinispan, ctx pipeline.Context) {
 				if ctx.IsTypeSupported(pipeline.ServiceMonitorGVK) {
 					i.ApplyMonitoringAnnotation()
 				}
+				// Ensure that CR has all the operator required labels
+				if !subMapOf(ctx.DefaultLabels(), i.ObjectMeta.Labels) || !subMapOf(ctx.DefaultAnnotations(), i.ObjectMeta.Annotations) {
```

Comment:
Do we need the `subMapOf` checks here given that we're only applying to new Infinispan CRs?

Can't we just revert this part of the code to how it was before your previous labels commit? i.e. just

```go
i.ApplyOperatorMeta(ctx.DefaultLabels(), ctx.DefaultAnnotations())
```